### ISSUE TITLE
How to explain AI to kids

### ISSUE THREAD

**Issue by yashsoni27** [swapagarwal/awesome-eli5#25]
Personally found this article to explain Artificial Intelligence to your kids and small beings who are keen to understand about the massive future technology
Consider adding it to your list... 
https://medium.com/eliza-effect/science-fiction-movie-trailers-and-youtube-videos-i-use-to-help-kids-understand-artificial-38a6c08d4652

**Comment by Tam-Mari** [comment]
Hello! I'd  love to make a contribution. I always try to break down difficult topics by using understandable language and funny examples.
